docs/registries.conf.5.md

```diff
@@ -0,0 +1,26 @@
+% registries.conf(5) System-wide registry configuration file
+% Brent Baude
+% Aug 2017
+
+# NAME
+registry.conf - Syntax of System Registry Configuration File
+
+# DESCRIPTION
+The REGISTRIES configuration file is a system-wide configuration file for container image
+registries. The file format is TOML.
+
+# FORMAT
+The [TOML_format][toml] is used to build simple list format for registries under two
+categories: `registries` and `insecure_registries`. You can list multiple registries using
```

Comment:
I know there's been several loops on this and I think additional, secure and registries have all been used.  Can/should we rename these (and possibly drive Brent completely bonkers) to 
public_registries and authenticated_registries?

I really, really don't like additional.  It's "additional" to what? Do you get docker.io by default if you don't specify anything in additional?  

My $.02, I won't harp, but did want to raise the thought.